modules.d/90dmsquash-live/dmsquash-live-root.sh

```diff
@@ -286,28 +287,49 @@ fi
 if [ -e /run/initramfs/live/${live_dir}/${squash_image} ]; then
     SQUASHED="/run/initramfs/live/${live_dir}/${squash_image}"
 fi
+if [ -e /run/initramfs/live/${live_dir}/${join} ]; then
+    SQUASHED_JOIN="/run/initramfs/live/${live_dir}/${join}"
+fi
 if [ -e "$SQUASHED" ]; then
     if [ -n "$live_ram" ]; then
         echo 'Copying live image to RAM...' > /dev/kmsg
         echo ' (this may take a minute)' > /dev/kmsg
         dd if=$SQUASHED of=/run/initramfs/squashed.img bs=512 2> /dev/null
         echo 'Done copying live image to RAM.' > /dev/kmsg
         SQUASHED="/run/initramfs/squashed.img"
+        if [ -n ${join} ];then
```

Comment:
```suggestion
        if [ -n ${join} ]; then
```
Did you intend to support the live_ram option?  If so, please verify that this new code path works.